main.tex

```diff
@@ -1241,16 +1344,16 @@ \subsection{Learn Astropy}
     \item[Tutorials] The \astropy tutorials are step-by-step demonstrations of
         common tasks that incorporate several packages or subpackages.
         Tutorials are more extended and comprehensive than examples, may contain
-        exercises for the users, and are generally geared towards workshops or
-        teaching.
+        exercises for the users, and are generally geared towards pedagogical
```

Comment:
Note: Just trying to be fancy at 3 AM. If you don't like this, I can change it back.

Reply:
Might be too fancy :wink:

Reply:
Do you want me to revert this? :sweat_smile:

Reply:
Sure